### PR TITLE
VZ-6718 Update for Jaeger tracing in Multicluster

### PR DESCRIPTION
**Pull request by ayaranga**

### CODE REVIEW

File: content/en/docs/monitoring/tracing/jaeger-tracing.md [hunk 1]
```diff
@@ -187,6 +187,10 @@ spec:
         "sidecar.jaegertracing.io/inject": "true"
 ```
 
+If you have multiple Jaeger instances in your cluster, specify the name of the Jaeger instance, that you intend to send
```
Comment (rosemarymarano) [inline content/en/docs/monitoring/tracing/jaeger-tracing.md:190]: instance, that -> instance that (no comma needed here)

File: content/en/docs/monitoring/tracing/jaeger-tracing.md [hunk 2]
```diff
@@ -257,4 +261,31 @@ storage:
     schedule: "55 23 * * *"                       // cron expression for it to run
 ```
 
+## Jaeger tracing in a Multicluster Verrazzano environment
```
Comment (rosemarymarano) [inline content/en/docs/monitoring/tracing/jaeger-tracing.md:264]: Multicluster  -> multicluster (lowercase)

File: content/en/docs/monitoring/tracing/jaeger-tracing.md [hunk 3]
```diff
@@ -257,4 +261,31 @@ storage:
...
+
+If Jaeger Operator component is enabled in the managed cluster, upon successful registration with the admin cluster,
```
Comment (rosemarymarano) [inline content/en/docs/monitoring/tracing/jaeger-tracing.md:266]: If Jaeger Operator -> If the Jaeger Operator
upon  -> after

File: content/en/docs/monitoring/tracing/jaeger-tracing.md [hunk 4]
```diff
@@ -257,4 +261,31 @@ storage:
...
+a Jaeger collector service runs in the managed cluster which exports the traces to the OpenSearch or Elasticsearch
```
Comment (rosemarymarano) [inline content/en/docs/monitoring/tracing/jaeger-tracing.md:267]: cluster which  -> cluster, which (comma needed here)

File: content/en/docs/monitoring/tracing/jaeger-tracing.md [hunk 5]
```diff
@@ -257,4 +261,31 @@ storage:
...
+NAME                                STATUS    VERSION   STRATEGY     STORAGE         AGE
+jaeger-verrazzano-managed-cluster   Running   1.34.1    production   elasticsearch   11m
+```
+
+### View the managed cluster traces
+
+You can see the managed cluster traces only from the Jaeger UI in the admin cluster. You can find the Jaeger UI URL for
+your admin cluster by following the instructions for [Accessing Verrazzano]({{< relref "/docs/access/_index.md" >}}).
+
+The spans include the Process tag `verrazzano_cluster` which has the name of the managed cluster. To see the traces
```
Comment (rosemarymarano) [inline content/en/docs/monitoring/tracing/jaeger-tracing.md:285]: tag `verrazzano_cluster` which -> tag `verrazzano_cluster`, which (always need a comma before "which")

File: content/en/docs/monitoring/tracing/jaeger-tracing.md [hunk 6]
```diff
@@ -257,4 +261,31 @@ storage:
...
+only for the managed cluster, search based on the tag `verrazzano_cluster=<managed cluster name>`.
```
Comment (rosemarymarano) [inline content/en/docs/monitoring/tracing/jaeger-tracing.md:286]: only for the managed cluster -> for the managed cluster only

**Review by rosemarymarano — COMMENTED** [review]
Few edits

File: content/en/docs/monitoring/tracing/jaeger-tracing.md [hunk 7]
```diff
@@ -257,4 +261,31 @@ storage:
     schedule: "55 23 * * *"                       // cron expression for it to run
 ```
 
+## Jaeger tracing in a Multicluster Verrazzano environment
+
+If Jaeger Operator component is enabled in the managed cluster, upon successful registration with the admin cluster,
+a Jaeger collector service runs in the managed cluster which exports the traces to the OpenSearch or Elasticsearch
+storage configured in the admin cluster.
+
+**Note:** - Traces are exported to the admin cluster only when the Jaeger instance in the admin cluster is configured
```
Comment (rosemarymarano) [inline content/en/docs/monitoring/tracing/jaeger-tracing.md:270]: **Note:** -  -> **NOTE**:  (bold the word NOTE (all capitals) only, then a colon and no hyphen.

File: content/en/docs/monitoring/tracing/jaeger-tracing.md [hunk 8]
```diff
@@ -257,4 +261,31 @@ storage:
...
+with the OpenSearch or Elasticsearch storage.
+
+Listing Jaeger resources in the managed cluster should show output similar to the following.
```
Comment (rosemarymarano) [inline content/en/docs/monitoring/tracing/jaeger-tracing.md:273]: should show -> shows

File: content/en/docs/monitoring/tracing/jaeger-tracing.md [hunk 9]
```diff
@@ -257,4 +261,31 @@ storage:
...
+```
+$ kubectl get jaegers -n verrazzano-monitoring
+NAME                                STATUS    VERSION   STRATEGY     STORAGE         AGE
+jaeger-verrazzano-managed-cluster   Running   1.34.1    production   elasticsearch   11m
+```
+
+### View the managed cluster traces
+
+You can see the managed cluster traces only from the Jaeger UI in the admin cluster. You can find the Jaeger UI URL for
```
Comment (rosemarymarano) [inline content/en/docs/monitoring/tracing/jaeger-tracing.md:282]: managed cluster traces only from the Jaeger UI in the admin cluster -> managed cluster traces from the Jaeger UI in the admin cluster only (the rule here is, "Position of only: Keep only close to that which it modifies to improve the clarity and accuracy of the sentence.") 

You can find the Jaeger UI URL for your admin cluster by following the instructions for -> suggested re-write -> To find the Jaeger UI URL for your admin cluster, follow the instructions for

**Review by rosemarymarano — COMMENTED** [review]
Few more edits and updated comments. If you have any questions, please ask me and I'll clarify.

**Review by rosemarymarano — APPROVED** [review]
Perfect, thank you.

**Review by fredtibbitts — APPROVED** [review]

**Pull request merged**